src/base/emu-i386/simx86/interp.c

```diff
@@ -241,56 +241,48 @@ static unsigned int _JumpGen(unsigned int P2, int mode, int cond,
 		    }
 		    /* forward jump or backward jump >=256 bytes */
 		    if (CONFIG_CPUSIM)
-			Gen(JF_LINK, mode, cond, P2, j_t, j_nt);
+			Gen(JF_LINK, mode, opc, P2, j_t, j_nt);
 		    else
-			Gen(JF_LINK, mode, cond, P2, j_t, j_nt, &InstrMeta[0].clink);
+			Gen(JF_LINK, mode, opc, P2, j_t, j_nt, &InstrMeta[0].clink);
 		}
 		break;
-	case 0x12: {   /* uncond jmp far */
+	case JMPld: {   /* uncond jmp far */
 		unsigned short jcs = FetchW(P2 + pskip - 2);
 		Gen(L_IMM, mode, Ofs_CS, jcs);
 		AddrGen(A_SR_SH4, mode, Ofs_CS, Ofs_XCS);
 	}
 	/* no break */
-	case 0x10:    /* uncond jmp */
+	case JMPsid: case JMPd:   /* uncond jmp */
 		if (dsp==0) {	// eb fe
 		    dbug_printf("!Forever loop!\n");
 		    leavedos_main(0xebfe);
 		}
-#if !defined(SINGLESTEP)
-		if (CONFIG_CPUSIM && !(EFLAGS & TF) && cond == 0x10) {
```

Comment:
Junk?
But its not change-logged.
OK though.

Reply:
yes I left that in accidentally, should really be in a separate commit.